src/NuGet.Core/NuGet.Packaging/ContentModel/ContentItemCollection.cs

```diff
@@ -46,12 +46,7 @@ public void Load(IEnumerable<string> paths)
             }
         }
 
-        public IEnumerable<ContentItem> FindItems(PatternSet definition)
-        {
-            return FindItemsImplementation(definition, _assets);
-        }
-
-        public IEnumerable<ContentItemGroup> FindItemGroups(PatternSet definition)
+        public void FindItemGroups(PatternSet definition, IList<ContentItemGroup> contentItemGroupList)
```

Comment:
Can you document it? 

Consider naming the parameter in a descriptive manner.

Reply:
@nkolev92 Could you add some detail so I know what you're referring to that could be documented and where?

The contentItemGroupList seems like a descriptive name to me, but if you have a suggestion on what it should be called instead, or why it's not descriptive, please let me know! Thanks!

Reply:
Created a new method instead and made this one obsolete and I've added docs for the new method based on your suggestion, thanks!